lib/active_model/validations/store_model_validator.rb

```diff
@@ -24,7 +24,7 @@ def validate_each(record, attribute, value)
         when :json
           strategy.call(attribute, record.errors, value.errors) if value.invalid?
         when :array
-          record.errors.add(attribute, :invalid) if value.any?(&:invalid?)
+          record.errors.add(attribute, :invalid) if value.select(&:invalid?).present?
```

Comment:
```suggestion
          record.errors.add(attribute, :invalid) if value.select(&:invalid?).any?
```
I guess we still can use `.any?` just to make the line shorter

Reply:
Personally I'd use `present?` as we care that the result is non-empty or not. Not much in it either way.

Reply:
In this particular case `.select` cannot return `nil` 🙂

Reply:
exactly 😁